docs/tutorials/other-resources/references.md

```diff
@@ -0,0 +1,61 @@
+# Additional References
+Below is a list of additional resources from the community.
```

Comment:
Most of them are our blogs, not from the community.

Reply:
yes, will add more community ones -- you have a point tho hahah

Reply:
We may want to have two sections:
- Additional References
   - All of the contents are from Aqua (official)
- Community Tutorials
  - All of the contents are from the community (unofficial)

Reply:
I renamed it to additional resources -- the thing is that official vs unofficial is going to be a bit scarce, and people would have to look at multiple places that have the same goal